src/main/java/com/kodilla/ecommercee/ServiceOrder.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.ecommercee;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+
+@Service
+public class ServiceOrder {
```

Comment:
Zapisujmy nazwy z Service  odwrotnie. Czyli OrderService zamiast ServiceOrder, bo ta druga nazwa sugeruje, że serwisujesz(obslugujesz) jakieś zamówienie, a nie, że to service do zamówien